data/EmmanuelAjibokun.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Emmanuel Ajibokun",
+  "bio": "Open Source Contributor || React Developer",
+  "links": [
+    {
+      "group": "Socials Primary",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/meet_manny",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
```

```suggestion
    }
```
The trailing comma isnot required